docs/presentations/linkedin-meetup-2024-04-24.qmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "Ibis"
+subtitle: "Portable Python DataFrames"
+author:
+  - Chloe He
+  - Cody Peterson
+  - Phillip Cloud
+format:
+  revealjs:
+    smaller: true
+    footer: <https://ibis-project.org>
+    preview-links: true
+    # https://quarto.org/docs/presentations/revealjs/themes.html#using-themes
+    theme: dark
+---
+
+# Who
+
+:::: {.columns}
+
+::: {.column width="50%"}
+Phillip Cloud
+
+- Principal engineer at Voltron Data
+- Python analytics for 10+ years
+- Open source
+- Tech lead for Ibis
+:::
+
+::: {.column width="50%"}
+Chloe He
+
+- …
+- …
+- …
+:::
+
+::::
+
+# What
+
+## Ibis is a Python frontend for:
+
+- Exploratory data analysis
+- General analytics
+- Data engineering
+- Machine learning
+- `$A_BUNCH_OF_OTHER_STUFF`
+
+# Examples
+
+## Examples
+
+<https://ibis-project.org/tutorials/getting_started>
+
+::: {.panel-tabset}
+
+## Raw data
+
+```{python}
+#| echo: true
+from ibis.interactive import *
+
+pengs = ex.penguins.fetch()
+pengs
```

Comment:
nit-y: I do think using explicit `ibis.examples` is better here for people likely new to Ibis

also would prefer `penguins` as the variable name, or just `p` if keeping it short